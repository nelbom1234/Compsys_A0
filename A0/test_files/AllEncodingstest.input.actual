test_files/AllEncodingstest.input: data
