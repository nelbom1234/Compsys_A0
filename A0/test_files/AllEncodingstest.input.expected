test_files/AllEncodingstest.input: Non-ISO extended-ASCII text
